HomeUI/src/views/apps/RegisterFluxApp.vue

```diff
@@ -1771,7 +1771,7 @@ export default {
       return countries;
     },
     regionsOptions(continentCode, countryCode, isNegative) {
-      const regions = [{ value: isNegative ? 'NONE' : 'ALL', text: isNegative ? 'NONE' : 'ALL' }];
+      const regions = [{ value: isNegative ? 'ALL' : 'ALL', text: isNegative ? 'ALL' : 'ALL' }];
```

Comment:
I think this was ok, on forbidden, if none is selected on continent it should say None not all. As it will not set anything if the it's on the default choice.